shared/linkify.ts

```diff
@@ -55,6 +55,27 @@ for (const schema of commonSchemes) {
 	linkify.add(schema + ":", "http:");
 }
 
+linkify.add("web+", {
+	validate(text: string, pos: number, self: LinkifyIt.LinkifyIt) {
+		const webSchemaRe = /^[a-z]+:/g;
```

Comment:
this should definitely be `i` - it's normalized by the browser. other than that the use of `g` and `lastIndex` looks fine...? not entirely sure, never used that before. (well, if the tests pass it's good right? .-.)

Reply:
I don't want to match case insensitive no. It should accept valid web+ schemes, not some other things.

g is needed that last index gets set

Reply:
it'll also accept `WEB+ap://example.org`. you can add a test for that if you don't believe it.

this should definitely have an `i`.

Reply:
ack